MaxText/sequence_packing.py

```diff
@@ -16,6 +16,9 @@
 
 """Packed Sequence Op."""
 
+# can we use this script out of the box for pytorch? if we want to stay away
+# from tf.data, we should explore further on how to repurpose this helper scrpit
```

Comment:
if we were to skip going through tf.data, how much work is repurposing this script for pytorch?

Reply:
This script essentially contains map operations for packing multiple rows into one until the length matches the max_sequence_length, This can be done on huggingface's datasets. Should be 3 days of work